### PR TITLE
feat(config): import Z-Wave Alliance Files batch 6

### PR DESCRIPTION
**Pull request by blhoward2**
Batch 6 for volunteer review

### CODE REVIEW

**Comment by scyto** [comment]
I got this batch

**Comment by scyto** [comment]
@blhoward2  first batch of 10 done - my core finding is the min and max are  more often wrong than not compared to the z-wave alliance page - in all cases i have matched that page - just wanted to be clear before anyone approved my first 10 :-)

**Comment by blhoward2** [comment]
@scyto When you're complete, check the failing tests. In this case lint has identified an error in the files...you included a comma in one of the values.

**Comment by scyto** [comment]
Ok, will do.

**Comment by scyto** [comment]
How good is the bot at fixing?

**Comment by scyto** [comment]
@blhoward2 you don’t have to fix my mistakes, I will work through it, best way for me to learn.

**Comment by scyto** [comment]
@blhoward2 you don’t have to fix my mistakes, I will work through it, best way for me to learn.

**Comment by blhoward2** [comment]
This set was so messed up to start it may be best for you to just look at them when I'm finished. Otherwise we're going to go back and forth a bunch.

**Comment by scyto** [comment]
Ok, I see you want us to be vicious on descriptions and labels and correct poor grammar and wording, I can do that. 
Some of the changes like units I get.
I fixed some of the basic errors like having removed a description in the device section, didn’t realize couldn’t do that and was following advice from parameter section.

on the switch you made massive edits not sure what to say

**Comment by scyto** [comment]
I don’t understand why some of the lint errors refer to files not in this PR?

**Comment by blhoward2** [comment]
@zwave-js-bot fix lint

**Comment by blhoward2** [comment]
@AlCalzone Ready for you

**Comment by blhoward2** [comment]
@AlCalzone in case you didn't see this one. It's ready.

File: packages/config/config/devices/0x017f/wink_d_w_sensor.json [hunk 1]
```diff
@@ -30,9 +30,9 @@
 			"allowManualEntry": true
 		},
 		"2": {
-			"label": "Basic_set Level",
-			"description": "1 to 99 dim level, 0 for off, 255 on",
+			"label": "Basic Set Level",
 			"valueSize": 1,
+			"unit": "%",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x017f/wink_d_w_sensor.json]: This shouldn't have a unit

File: packages/config/config/devices/0x0166/sw-zrc.json [hunk 2]
```diff
@@ -0,0 +1,321 @@
...
+					"value": 1
+				},
+				{
+					"label": "Association Group 2",
+					"value": 2
+				}
+			]
+		},
+		"2": {
+			"label": "Function of Remswiid™ Device Key A",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x0166/sw-zrc.json]:
```suggestion
			"label": "Function of Device Key A",
```

File: packages/config/config/devices/0x0166/sw-zrc.json [hunk 3]
```diff
@@ -0,0 +1,321 @@
...
+					"value": 2
+				},
+				{
+					"label": "Control Scene Activation 1",
+					"value": 3
+				}
+			]
+		},
+		"3": {
+			"label": "Function of Remswiid™ Device Key B",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x0166/sw-zrc.json]:
```suggestion
			"label": "Function of Device Key B",
```

Comment (AlCalzone) [reply packages/config/config/devices/0x0166/sw-zrc.json]: etc...

File: packages/config/config/devices/0x016a/ft102.json [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+					"value": 0
+				},
+				{
+					"label": "Hail",
+					"value": 1
+				}
+			]
+		},
+		"255": {
+			"label": "Reset Your Touch",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x016a/ft102.json]:
```suggestion
			"label": "Factory Reset",
```

File: packages/config/config/devices/0x016a/ft112-k.json [hunk 5]
```diff
@@ -0,0 +1,160 @@
...
+					"value": 0
+				},
+				{
+					"label": "Lock",
+					"value": 1
+				}
+			]
+		},
+		"255": {
+			"label": "Reset The Sensor",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x016a/ft112-k.json]: We should make sure to name the factory reset params similarly

File: packages/config/config/devices/0x016a/ft130-k.json [hunk 6]
```diff
@@ -0,0 +1,270 @@
...
+			"minValue": 0,
+			"maxValue": 1,
+			"defaultValue": 0,
+			"unsigned": true,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Baisc set",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x016a/ft130-k.json]:
```suggestion
					"label": "Basic set",
```

File: packages/config/config/devices/0x016a/ufa01.json [hunk 7]
```diff
@@ -0,0 +1,223 @@
+// Fantem UFA01
+// KASAIHOUCHIKIADAPTOR01
+{
+	"manufacturer": "Fantem",
+	"manufacturerId": "0x016a",
+	"label": "UFA01",
+	"description": "KASAIHOUCHIKIADAPTOR01",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x016a/ufa01.json]: WTF 😱

File: packages/config/config/devices/0x016a/ufa01.json [hunk 8]
```diff
@@ -0,0 +1,223 @@
...
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Send 0xff when on, 0x00 when off",
+					"value": 0
+				},
+				{
+					"label": "Send 0x00 when on, 0xff when off",
+					"value": 1
+				}
```
Comment (AlCalzone) [inline packages/config/config/devices/0x016a/ufa01.json]: Should we mention the decimal values 0/255 instead?

Comment (AlCalzone) [reply packages/config/config/devices/0x016a/ufa01.json]: same for the other parameters

File: packages/config/config/devices/0x016a/ufa01.json [hunk 9]
```diff
@@ -0,0 +1,223 @@
...
+				},
+				{
+					"label": "Basic Set",
+					"value": 256
+				},
+				{
+					"label": "Binary report and Basic Set",
+					"value": 272
+				}
+			]
```
Comment (AlCalzone) [inline packages/config/config/devices/0x016a/ufa01.json]: Looks like partial parameters, but I guess this is okay

File: packages/config/config/devices/0x0173/lgrm.json [hunk 10]
```diff
@@ -0,0 +1,75 @@
...
+				{
+					"label": "Disabled",
+					"value": 0
+				}
+			]
+		},
+		"3": {
+			"label": "Report Rate Interval (Association Group 1)",
+			"valueSize": 1,
+			"unit": "6 minutes",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x0173/lgrm.json]: That is a weird unit. Is that correct?

File: packages/config/config/devices/0x0175/mt_2653.json [hunk 11]
```diff
@@ -0,0 +1,387 @@
...
+					"value": 0
+				},
+				{
+					"label": "Yes",
+					"value": 1
+				}
+			]
+		},
+		"25": {
+			"label": "Blocks Wakeups",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x0175/mt_2653.json]:
```suggestion
			"label": "Block Wakeups",
```

File: packages/config/config/devices/0x0178/9125051.json [hunk 12]
```diff
@@ -0,0 +1,81 @@
...
+			"options": [
+				{
+					"label": "Enable",
+					"value": 1
+				},
+				{
+					"label": "Disable",
+					"value": 0
+				}
+			]
```
Comment (AlCalzone) [inline packages/config/config/devices/0x0178/9125051.json]: I think the additional values mentioned in the description should have an option (maybe with a `(readonly)` in the label)

File: packages/config/config/devices/0x017f/wink_d_w_sensor.json [hunk 13]
```diff
@@ -0,0 +1,61 @@
...
+			"minValue": 0,
+			"maxValue": 255,
+			"defaultValue": 99,
+			"unsigned": true,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": true,
+			"options": [
+				{
+					"label": "Off/cancel alarm/etc",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x017f/wink_d_w_sensor.json]:
```suggestion
					"label": "Off",
```

File: packages/config/config/devices/0x017f/wink_d_w_sensor.json [hunk 14]
```diff
@@ -0,0 +1,61 @@
...
+					"value": 0
+				},
+				{
+					"label": "On (for binary switch)",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x017f/wink_d_w_sensor.json]:
```suggestion
					"label": "On",
```

File: packages/config/config/devices/0x017f/wink_motion_sensor.json [hunk 15]
```diff
@@ -0,0 +1,183 @@
...
+			"defaultValue": 30,
+			"unsigned": true,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": true
+		},
+		"3": {
+			"label": "Basic Set Level",
+			"valueSize": 1,
+			"unit": "%",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x017f/wink_motion_sensor.json]:
```suggestion
```

File: packages/config/config/devices/0x017f/wink_motion_sensor.json [hunk 16]
```diff
@@ -0,0 +1,183 @@
...
+			"minValue": 0,
+			"maxValue": 255,
+			"defaultValue": 255,
+			"unsigned": true,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": true,
+			"options": [
+				{
+					"label": "Turn off",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x017f/wink_motion_sensor.json]:
```suggestion
					"label": "Off",
```
Please make these all similar

**Review by AlCalzone — CHANGES_REQUESTED** [review]

File: packages/config/config/devices/0x0166/sw-zrc.json [hunk 17]
```diff
@@ -69,7 +69,7 @@
 			]
 		},
 		"3": {
-			"label": "Function of Remswiid™ Device Key B",
+			"label": "Function of Device Key B",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x0166/sw-zrc.json]: Please apply this change to the other parameters below too

File: packages/config/config/devices/0x016a/ft112-k.json [hunk 18]
```diff
@@ -95,15 +95,15 @@
...
+					"label": "Basic set",
 					"value": 1
 				},
 				{
 					"label": "Sensor binary report",
 					"value": 2
 				},
 				{
-					"label": "Basic Set and sensor binary report",
+					"label": "Basic set and sensor binary report",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x016a/ft112-k.json]: One could argue that these are the CC names, so `Basic Set` and `Sensor Binary Report` would be better. In any case, they should be consistent.

File: packages/config/config/devices/0x0173/lgrm.json [hunk 19]
```diff
@@ -50,7 +50,7 @@
 		"3": {
 			"label": "Report Rate Interval (Association Group 1)",
 			"valueSize": 1,
-			"unit": "6 minutes",
+			"unit": "6 min per count",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x0173/lgrm.json]: That's not actually better. Is it really the configured value times 6? In that case, the original was okay.
Or: `x6 minutes` 🤷🏻‍♂️

File: packages/config/config/devices/0x0178/9125051.json [hunk 20]
```diff
@@ -30,23 +30,31 @@
 		},
 		"2": {
 			"label": "Discovery Mode",
-			"description": "Start, stop, or get the results of discovery mode. Retrieve the configuration parameter to obtain the results: 0=not enabled, 1=enabled, 2=timed out, 3=failure",
+			"description": "Start, stop, or get the results of discovery mode. Retrieve the configuration parameter to obtain the results: 0=disabled, 1=enabled, 2=Timeout, 3=Failure",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x0178/9125051.json]:
```suggestion
			"description": "Start, stop, or get the results of discovery mode. Retrieve the configuration parameter to obtain the results.",
```
We have the labels in the options, no need to repeat them.

File: packages/config/config/devices/0x0178/9125051.json [hunk 21]
```diff
@@ -30,23 +30,31 @@
...
+					"value": 0
+				},
 				{
 					"label": "Enable",
 					"value": 1
 				},
 				{
-					"label": "Disable",
-					"value": 0
+					"label": "Timeout (readOnly)",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x0178/9125051.json]:
```suggestion
					"label": "Timeout (readonly)",
```

File: packages/config/config/devices/0x0178/9125051.json [hunk 22]
```diff
@@ -30,23 +30,31 @@
...
+					"value": 2
+				},
+				{
+					"label": "Failure (readOnly)",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x0178/9125051.json]:
```suggestion
					"label": "Failure (readonly)",
```

**Review by AlCalzone — CHANGES_REQUESTED** [review]
Sorry, I feel like I'm making you run in circles but now's our chance to make the config files consistent.

File: packages/config/config/devices/0x0173/lgrm.json (hunk 19 above)
Comment (Technikte) [reply packages/config/config/devices/0x0173/lgrm.json]: The value is 1/10th of an hour, so 1 means 6 minuntes... 1/10x60xValue = eg. 6 minutes.

Comment (AlCalzone) [reply packages/config/config/devices/0x0173/lgrm.json]: Okay then lets stick with `6 minutes` i guess.

**Comment by Technikte** [comment]
I hope it is ready now. Next time I will try to spend more time when doing the work.

**Review by AlCalzone — DISMISSED** [review]

**Pull request merged**